src/api/parser/src/utils/storage.js

```diff
@@ -1,6 +1,10 @@
-const { logger, Redis } = require('@senecacdot/satellite');
+const { logger } = require('@senecacdot/satellite');
+const Redis = require('../lib/redis');
```

Comment:
Get this from Satellite

Reply:
The Redis exported from `../lib/redis` is imported from Satellite

Reply:
What's the value of that extra indirection?  You're not doing anything with it.  Just use Satellite's `Redis` everywhere you need it.

We used to have a module for this because we did setup for you.  Now we don't.

Reply:
I'll remove all the indirections